lib/ranked-model.rb

```diff
@@ -65,6 +65,16 @@ def ranks *args
         ranker.with(self).relative_rank
       end
 
+      define_method "#{ranker.name}_insert_after!" do |previous|
+        if previous == '-1'
```

Comment:
Also, I'm not sure about the `-1`. How would we come across that as a value?

Reply:
In case if we want to move the object to the top of the list, there will be no `previous_record` in the list and so we can set the expectation, that in case if `-1` is passed, the current record would be moved to the top of the list.

Reply:
You could just not pass a value for `previous` and check if it doesn't exist: `unless previous`. :)

Reply:
Hmm yes. I just thought it would be more explicit to set a specific value. But yea, I think checking if it exists should suffice.